src/routes/pdf-fetcher/containers/PdfFetcherPage.js

```diff
@@ -71,7 +71,7 @@ class PdfFetcherPage extends React.Component {
     const children = this.props.categories.getChildren(category)
     const toc = isEmpty(children)
 
-    this.setState(Object.assign({}, this.state, {loading: category}))
+    this.setState((prevState) => Object.assign({}, prevState, {loading: category}))
```

Comment:
Is there a linter rule to prefer `{...prevState, loading: category}` over `Object.assign({}, prevState, {loading: category})`? Probably not... 🤔

Reply:
You forget that npm exists :P https://www.npmjs.com/package/eslint-plugin-prefer-object-spread

Reply:
Whoops silly me :D